antlr/JSONFormula.g4

```diff
@@ -34,7 +35,7 @@ expression
   | literal # literalExpression
   | functionExpression # functionCallExpression
   | expression '|' expression # pipeExpression
-  | RAW_STRING # rawStringExpression
+  | (STRING | RAW_STRING) # rawStringExpression
```

Comment:
Do we want RAW_STRING and STRING to be separate ? The difference between them is inside JSON literals only a few characters can be escaped while outside all characters are.
```
`"\\z"` is not valid
"\\z" is valid
```

Reply:
And the above is true as per antlr grammar. In our Parser, we support both of them.

Reply:
Hmm.  It appears that not all characters can be escaped?
`JSON.parse('"\\z"')` results in an error. 
Whereas 
`JSON.parse('"\\""')` or `JSON.parse('"\\n"')` succeed.

Reply:
So maybe our parser should be more strict?

Reply:
https://datatracker.ietf.org/doc/html/rfc8259#section-7

Reply:
https://github.com/adobe/json-formula/issues/70